docker-compose.tenant.yml

```diff
@@ -22,11 +22,13 @@ services:
                     - mongo-tenantadm
 
     mender-conductor-workers:
-        image: mendersoftware/conductor-workers:master
+        image: workers-temp
```

Comment:
Did you mean to change this to `workers-temp`?

Reply:
absolutely not, fixed